devices/FreeEmgWearableDevice/applications/FREEEMG_Log.xml

```diff
@@ -19,7 +19,7 @@
 
     <module>
         <name>yarprobotinterface</name>
-        <parameters>--config freeemg_logger.xml --loggerType "(matlab)" --period 0.01 --experiment-name free_emg_data --experiment-dir C:/tmp/ifeel_data/ </parameters>
+        <parameters>--config freeemg_logger.xml --loggerType "(matlab)" --period 0.01 --experiment-name free_emg_data --path C:/tmp/ifeel_data/ </parameters>
```

Comment:
```suggestion
        <parameters>--config freeemg_logger.xml --loggerType "(matlab)" --period 0.01 --experiment-name free_emg_data --path C:/tmp/data/ </parameters>
```
Probably it makes sense to change the **default** directory path this way 👀 .

cc @lrapetti